packages/modules-sdk/src/definitions.ts

```diff
@@ -118,6 +121,20 @@ export const ModulesDefinition: { [key: string | Modules]: ModuleDefinition } =
         resources: MODULE_RESOURCE_TYPE.SHARED,
       },
     },
+    [Modules.PROMOTION]: {
+      key: Modules.PROMOTION,
+      registrationName: ModuleRegistrationName.PROMOTION,
+      defaultPackage: false,
+      label: "promotionModuleService",
```

Comment:
```suggestion
      label: "PromotionModuleService",
```

label seems to be capitalized, but registration name isn't. Do you know what this is used for?

Reply:
you can use `upperCaseFirst` on `ModuleRegistrationName.PROMOTION` wdyt?